docs/src/pages/styles/basics/basics.md

```diff
@@ -129,6 +129,210 @@ export default withStyles(styles)(HigherOrderComponent);
 
 {{"demo": "pages/styles/basics/HigherOrderComponent.js"}}
 
+#### TypeScript usage
```

Comment:
I would have personally host it under the advanced page to keep the basic page focused on the introduction. I would have also rendered it as a h2 so it gets its own entry in the TOCs and hash link. But if you think it's better as is, happy too.

Reply:
Agree, let me update.